tests/http_server/test_http_server.py

```
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+"""Test script for the Trakt MCP HTTP server."""
+
+import requests
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)